ingest/ledgerbackend/stellar_core_runner.go

```diff
@@ -42,6 +44,9 @@ type stellarCoreRunner struct {
 	metaPipe    io.Reader
 	tempDir     string
 	nonce       string
+
+	// Optionally, logging can be done to something other than stdout.
+	Log *log.Entry
```

Comment:
Let's make it private as other fields.
```suggestion
	log *log.Entry
```